staging/src/k8s.io/component-base/metrics/counter_test.go

```diff
@@ -19,6 +19,7 @@ package metrics
 import (
 	"bytes"
 	"testing"
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: Imports should be grouped. standard libs -> third party -> k8s imports

Reply:
+1

Reply:
@conwaychriscosmo, as @RainbowMango mentioned it's conventional in the K8s project to separate each group of imports with a blank line. In this case, the assert lib should be in the middle group, along with semver and expfmt.